README.md

```diff
@@ -1,62 +1,55 @@
 
 ![Node.js CI](https://github.com/openmrs/openmrs-esm-patient-management/workflows/Node.js%20CI/badge.svg)
 
-
 # OpenMRS Patient Management
 
-This is a [Lerna](https://lerna.js.org/) project containing patient management frontend modules which deal with creating, searching, and listing patients. This package handles the following:
+This repository contains frontend modules for the OpenMRS SPA. These modules relate to registering and editing patients, searching for existing patients, creating and managing patient lists and managing patient queues in an outpatient setting. The modules within this repository include:
 
--  [@openmrs/esm-patient-search-app](packages/esm-patient-search-app)
--  [@openmrs/esm-patient-registration-app](packages/esm-patient-registration-app)
--  [@openmrs/esm-patient-list-app](packages/esm-patient-list-app)
--  [@openmrs/esm-outpatient-app](packages/esm-outpatient-app)
+- [Active visits app](packages/esm-active-visits-app/)
+- [Outpatient app](packages/esm-outpatient-app/README.md)
+- [Patient search](packages/esm-patient-search-app)
+- [Patient registration](packages/esm-patient-registration-app)
+- [Patient list](packages/esm-patient-list-app)
 
-## Repository Development
+## Setup
 
+Check out the developer documentation [here](http://o3-dev.docs.openmrs.org).
 
-### Prerequisites
+This monorepo uses [yarn](https://yarnpkg.com) and [lerna](https://github.com/lerna/lerna).
 
-- [Node](https://nodejs.org/en/download)
-- yarn ```sh npm install yarn -g ```
-### Getting started
+To start a dev server running all the modules simultaneously, run:
 
-To install and setup the repository once cloned just use the following command
-
-```sh
-npx lerna bootstrap
+```bash
+yarn start
 ```
 
-To develop on a specific package e.g [@openmrs/esm-patient-search-app](packages/esm-patient-search-app)
+This command uses the [openmrs](https://www.npmjs.com/package/openmrs) tooling to fire up a dev server running `esm-patient-chart` as well as the specified module.
 
-```sh
-npx openmrs develop --sources 'packages/esm-patient-search-app'
-```
+Note that this is very resource-intensive.
 
-You can always use regex
+To start a dev server for a specific module, run:
 
-```sh
-npx openmrs develop --sources 'packages/esm-patient-{package1,package2}-app/'
+```bash
+yarn start --sources 'packages/esm-<insert-package-name>-app'
 ```
 
+You could provide `yarn start` with as many `sources` arguments as you require. For example, to run the patient registration and patient search modules only, use:
 
-### Building
-
-For building the code just run
-
-```sh
-npx lerna run build
-
+```bash
+yarn start --sources 'packages/esm-patient-search-app' --sources 'packages/esm-patient-registration-app'
```

Comment:
Something's gone wrong here... doesn't `yarn start` fire up all the packages?

Reply:
It does.

Reply:
Right so 

> dev server running `esm-patient-chart` as well as the specified module.

doesn't apply; and with

> `yarn start --sources 'packages/esm-<insert-package-name>-app'

there's no reason you would use `--sources`.

Reply:
The first point is correct.

You would use `--sources` to run a specific module (or even multiple modules) locally.